examples/aws/terraform/ha-autoscale-cluster/ansible/upgrade.yaml

```diff
@@ -23,43 +23,43 @@
     - name: Copy binaries from local directory
       when: teleport_path is defined
       block:
-      - copy:
+        - copy:
```

Comment:
Indentation is funky here. We should either outdent the `copy` or indent the `src` and `dest` for clarity IMO.

Reply:
I did the latter. `ansible-lint` still passes.